### ISSUE TITLE
[Bug]: More "Recently edited" files are accessible via keyboard than by mouse 

### ISSUE THREAD

**Issue by nimishavijay** [nextcloud/server#34088]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

When you are tabbing through Files, on reaching the "Recently edited" files, only 3 files are visible and accessible via mouse but there more files (at least 4 more in my case) that can be reached by tabbing with the keyboard

### Steps to reproduce

1. Open Files
2. Tab through main content until "Recently edited" files are reached
3. Keep tabbing 
4. Even though there are only 3 files visible, on tabbing you find that there are more files below but hidden
![image](https://user-images.githubusercontent.com/52440189/190391232-fbf496df-ab61-492c-840a-97f28a6ae969.png)

### Expected behavior

After the 3 visible files are focused, the focus shifts to the checkbox on the file list header row.

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
/
```

**Comment by jancborchardt** [comment]
Yep, we should make sure that all which are not visible are actually not rendered or set to `display: none;`.

On desktop we show 1 row only, but on mobile we show 3 rows of 2 entries each.

**Comment by PVince81** [comment]
First we need to disable scrolling so that even if someone tabs there, it doesn't scroll to the hidden items

Second, we need to find a way to detect which items are on the second row and not visible. If there's no plugin to do this (in Vue?) one can compare every element's "offsetTop" value with the "scrollTop + scrollHeight" of the container (or something similar).
For each element not visible, set their CSS to "display: none" to make sure they are not tab-abble.
